packages/@react-aria/overlays/src/usePopover.ts

```diff
@@ -36,14 +38,23 @@ export interface AriaPopoverProps extends Omit<PositionProps, 'isOpen'> {
    * reader experience. Only use with components such as combobox, which are designed
    * to handle this situation carefully.
    */
-  isNonModal?: boolean
+  isNonModal?: boolean,
+  /**
+   * Whether pressing the escape key to close the popover should be disabled.
+   * @default false
+   */
+  isKeyboardDismissDisabled?: boolean
```

Comment:
Not sure we really want this for popovers, but it was previously supported by DialogTrigger...